python/phylanx/ast/physl.py

```diff
@@ -389,6 +401,13 @@ def _Call(self, node):
                 return [symbol, ('1', [op, args])]
             else:
                 return [symbol, ('1', args)]
+        elif 'full' in symbol:
+            symbol = symbol.replace('full', 'constant' + dtype)
+            op = get_symbol_info(node.func, 'list')
+            if isinstance(args[0], tuple):
+                return [symbol, (args[1], [op, args[0]])]
+            else:
+                return [symbol, (args[1], args[0])]
```

Comment:
This might not belong into this PR, however `vstack()` should obey any given `dtype` as well.

Reply:
It is already supported, please see [here](https://github.com/STEllAR-GROUP/phylanx/blob/f4e0fb1f46c8c93c511bd69c61c6bcf72cf378d8/python/phylanx/ast/physl.py#L367).

Reply:
dtype support for vstack will be added by a separate PR.